bee-node/src/config.rs

```diff
@@ -17,10 +17,24 @@ use bee_snapshot::{SnapshotConfig, SnapshotConfigBuilder};
 
 use serde::Deserialize;
 
-pub(crate) const CONFIG_PATH: &str = "./config.toml";
+use std::fs;
+
+const CONFIG_PATH: &str = "./config.toml";
+
+// TODO use proper error
+/// Creates a Bee config builder from the local config file.
+pub fn read_config() -> Result<NodeConfigBuilder, ()> {
```

Comment:
Should this be a method of `NodeConfigBuilder` ? Like a `::from_file() -> Result<Self, ()>` ?

Reply:
yea, sounds good :+1: